### PR TITLE
feat: Up next button with node

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

Ensures that the up next button will work with the `Node` interfaces. 

Also added stories :) 

Screen Shot 2020-09-09 at 10 40 41 AM

### What design trade-offs/decisions were made?

This was a simple PR. the only controversial choice (in my mind) was allowing the component to support either `nodeId` or `componentId` 

### How do I test this PR?

I added two stories :D 

### What automated tests did you write?

New test over a new possible data flow for the button. 

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/ConnectedHorizontalFeed.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
```
Comment (redreceipt) [inline packages/apollos-ui-connected/src/ConnectedHorizontalFeed.js:1]: don't think you meant to leave this file in there

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/ConnectedHorizontalFeed.js:1]: I merged from the web UI and I guess this was left behind... whoops.

File: packages/apollos-ui-fragments/src/content.js [hunk 2]
```diff
@@ -2,6 +2,20 @@ import gql from 'graphql-tag';
 
 const CONTENT_UP_NEXT_FRAGMENT = gql`
   fragment ContentUpNextFragment on ContentItem {
+    ... on ProgressNode {
+      upNext {
+        id
+      }
+    }
+    ... on ContentParentNode {
```
Comment (redreceipt) [inline packages/apollos-ui-fragments/src/content.js:10]: don't you also need to take into account `ContentChildNode` cases where the up next button is on a child content item?

Comment (vinnyjth) [reply packages/apollos-ui-fragments/src/content.js:10]: We currently don't render the button on children.

Comment (redreceipt) [reply packages/apollos-ui-fragments/src/content.js:10]: ah

**Review by redreceipt — APPROVED** [review]

**Pull request merged**